pkg/controller/tas/resource_flavor.go

```diff
@@ -161,7 +161,7 @@ func (h *topologyHandler) Create(ctx context.Context, e event.CreateEvent, q wor
 		return
 	}
 
-	defer h.notifyTopologyWatchers(nil, topology)
+	defer h.queues.NotifyTopologyUpdateWatchers(nil, topology)
```

Comment:
We could move the `topologyHandler` to the topology_controller.go, but it can be in a follow up. I see it would mean more changes.